recipe/meta.yaml

```diff
@@ -8,37 +8,40 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/starsessions-{{ version }}.tar.gz
-  sha256: cb250de84ebc6159ad187cab69e6fe60eab11684b40349457e74dcfb7656c805
+  sha256: 8d3b509d4e6d235655f7dd495fcf0afc1bd86da84de3a8d434e6f82137ebcde8
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps -vv
 
 requirements:
   host:
     - pip
     - poetry-core >=1.0.0
-    - python >=3.8.0,<4.0.0
+    - python
+    - wheel
```

Comment:
```suggestion
```
Not needed when poetry is used. Actually, `wheel` should be needed only when `setuptools` is used.